packages/spark/base/_blockquotes.scss

```diff
@@ -0,0 +1,24 @@
+// ==========================================================================
+// Blockquote
+// ==========================================================================
+// Credit goes to our very own Dave on https://github.com/ScalesCSS/scalescss/tree/master/packages/base-blockquotes
+.sprk-b-Blockquote {
```

Comment:
All these files are pretty much a direct port from scales as they are written. Any sorts of refactors we want to do, such as Flag (we know we've wanted to redo this) should be done in a follow up issue. They will be much easier to do now as well since we don't have to modify the scalescss package.